src/main/resources/theme/chat/chat_section.html

```diff
@@ -1,7 +1,10 @@
 <div id="chat-section-{section-id}" class="chat-section chat-section-{username}">
   <div class="chat-section-head">
     <img class="avatar" src="{avatar}"/>
-    <img class="country-flag" src="{country-flag}"/>
+    <img class="country-flag" src="{country-flag}"
+      onmouseover="showCountryInfo(this)"
```

Comment:
This should rather be `showCountryInfo('{country}')` (which is a placeholder that first needs to be implemented)